src/cmd/backup-restore-tachyon.js

```diff
@@ -37,23 +37,28 @@ module.exports = class BackupRestoreTachyonCommand extends CLICommandBase {
 			this.firehosePath,
 			xmlFile
 		];
-
+		const startTime = new Date();
+		const outputLog = path.join(logDir, `tachyon_${deviceId}_backup_${Date.now()}.log`);
+		addLogHeaders({ outputLog, startTime, deviceId, commandName: 'Tachyon backup' });
+		this.ui.stdout.write(`Backing up NV data from device ${deviceId}...${os.EOL}`);
```

Comment:
This message is duplicated above (line 30). Also there are now 2 log files: one for the gpt read and one for the backup. Can you combine them into one?